src/AppContainer.prod.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+
+'use strict';
+
+const React = require('react');
+const { Component } = React;
+
+class AppContainer extends Component {
+  render() {
+    if (this.props.component) {
+      return <this.props.component {...this.props.prop} />;
```

Comment:
Is it worth dropping the `component` prop and only having `children` before this goes into a final release perhaps?